src/Mod/Spreadsheet/App/PropertySheet.cpp

```diff
@@ -2267,3 +2268,52 @@ bool PropertySheet::hasSpan() const
 {
     return !mergedCells.empty();
 }
+
+void PropertySheet::getLinksTo(std::vector<App::ObjectIdentifier>& identifiers,
+                               App::DocumentObject* obj,
+                               const char* subname,
+                               bool all) const
+{
+    Expression::DepOption option =
+        all ? Expression::DepOption::DepAll : Expression::DepOption::DepNormal;
+
+    App::SubObjectT objT(obj, subname);
+    auto subObject = objT.getSubObject();
+    auto subElement = objT.getOldElementName();
+
+    auto owner = Base::freecad_dynamic_cast<App::DocumentObject>(getContainer());
+    for (const auto& [cellName, cellExpression] : data) {
+        if (auto expr = cellExpression->getExpression()) {
+            const auto& deps = expr->getDeps(option);
+            auto it = deps.find(obj);
+            if (it == deps.end()) {
+                continue;
+            }
+            const auto [docObj, depsList] = *it;
+            for (auto& [depName, paths] : depsList) {
+                if (!subname) {
+                    identifiers.emplace_back(owner, cellName.toString().c_str());
+                    break;
+                }
+                bool found = false;
```

Comment:
Use std::any_of

Reply:
Done, although I don't think it's the only place.  I am actively avoiding the complete PropertyLinks.cpp and associated refactoring not because it doesn't desperately need it, but I'm trying to keep blinders on and focus on getting the TNP mitigation package across.

There's lots of post-transfer stuff that needs addressing, and would be a breath of fresh air.

Reply:
Done.  Likely not the only place :(.  Trying to avoid getting sucked into refactoring PropertyLinks.cpp and everything associated with it until after we get the TNP mitigation stuff stable.

Good note, I'm just cranky today.